### DISCUSSION TITLE
Does this have better generation and upscale speed than AUTO1111 and Forge?

### DISCUSSION THREAD

**Discussion by koopa990** [Stability-AI/StableSwarmUI#356] [Q&A]
Im curious. 

NVIDIA 2070

**Comment by mcmonkey4eva** [comment]
Than auto: Yes
Than Forge: depends, but probably about equivalent.

Swarm uses a Comfy backend, and Forge uses a copypaste of comfy code as its backend, so they should be about the same

**Reply by koopa990** [reply]
I am going through a crisis-- where I bricked my Forge, unable to use it. Then I went back to auto1111, which is the same old crappy speed before I switched to Forge. Is there a settting I can change for better perfomance?

Or I can check out Stable Swarm?

**Reply by mcmonkey4eva** [reply]
I'm biased but you should definitely use Swarm

**Reply by koopa990** [reply]
> I'm biased but you should definitely use Swarm

I want to be absolutely sure that swarm will have extensions support? and other stuff that may not brick my local Swarm install?

**Reply by mcmonkey4eva** [reply]
Swarm has its own extension support (but not many extensions available currently), and has Comfy extension support (of which there are many)
Swarm is pretty recoverable if you muck anything up, plus I'm available to help whenever you need it here on github or on the [discord]( https://discord.gg/q2y38cqjNw)

**Reply by koopa990** [reply]
> Swarm has its own extension support (but not many extensions available currently), and has Comfy extension support (of which there are many) Swarm is pretty recoverable if you muck anything up, plus I'm available to help whenever you need it here on github or on the [discord](https://discord.gg/q2y38cqjNw)

do you  know the install size for Stable Swarm?

**Reply by yoinked-h** [reply]
> > Swarm has its own extension support (but not many extensions available currently), and has Comfy extension support (of which there are many) Swarm is pretty recoverable if you muck anything up, plus I'm available to help whenever you need it here on github or on the [discord](https://discord.gg/q2y38cqjNw)
> 
> do you know the install size for Stable Swarm?

im on linux, but windows should be a bit larger (python embedded)
- comfyUI: 15gb (mostly venv)
- swarm: <1gb

**Reply by koopa990** [reply]
> > > Swarm has its own extension support (but not many extensions available currently), and has Comfy extension support (of which there are many) Swarm is pretty recoverable if you muck anything up, plus I'm available to help whenever you need it here on github or on the [discord](https://discord.gg/q2y38cqjNw)
> > 
> > 
> > do you know the install size for Stable Swarm?
> 
> im on linux, but windows should be a bit larger (python embedded)
> 
>     * comfyUI: 15gb (mostly venv)
> 
>     * swarm: <1gb

less than 1GB?? Is that real? or Is it actually 16GB

**Reply by mcmonkey4eva** [reply]
The comfy backend is generally included with swarm, so the total usage would be somewhere in the range of 10-15 gigs.  The core of swarm itself is basically nothing
